server/routes/v1/user.ts

```diff
@@ -129,11 +130,21 @@ async function resetPassHandler(req: Request, res: Response) {
   }
 }
 
+async function me(req: Request, res: Response) {
+  try {
+    const user = getAuthUser(res);
+    res.status(200).json(user);
+  } catch (err) {
+    console.error(err);
```

Comment:
Respond with an error message/unauthorized?

Reply:
`getAuthUser` does.
https://github.com/GDSCUTM-CommunityProjects/UBoard/blob/main/server/middleware/auth.ts#L23

Reply:
Should we have any tests for this, since there doesnt seem to be any for getauthuser? Other than that it looks good

Reply:
Really this type of a test would be: 1/ set the user on locals 2/ assert that it was set (getAuthUser returns the user). So I don't really think it's worth testing a setter/getter. LMKWYT!

It is Indirectly tested in other places as well.

Reply:
Fair enough. I think it's good to merge then.